applications/callflow/src/module/cf_call_forward.erl

```diff
@@ -22,7 +22,7 @@
 -define(MOD_CONFIG_CAT, <<(?CF_CONFIG_CAT)/binary, ".call_forward">>).
 -define(KEY_LENGTH, 1).
 
--define(MIN_CALLFWD_NUMBER_LENGTH, kapps_config:get_integer(?MOD_CONFIG_CAT, <<"min_callfwd_number_length">>, 3)).
+-define(MIN_CALLFWD_NUMBER_LENGTH, max(kapps_config:get_integer(?MOD_CONFIG_CAT, <<"min_callfwd_number_length">>, 3), 1)).
```

Comment:
Why floor the config to 1? While I appreciate the guard rails, configs generally are foot guns if used improperly :)

Reply:
@jamesaimonetti I was matching the minimum length of 1 that is being added to `kzd_users`. Let me know if you'd prefer I omit this part.

Reply:
no its fine, was just curious.